### DISCUSSION TITLE
Star != Follow

### DISCUSSION THREAD

**Discussion by kassner** [community/community#66305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It seems the new feed is mixing up star with follow (or even the watch feature). For example:

![Screenshot from 2023-09-07 09-46-12](https://github.com/community/community/assets/301255/05ef1263-9479-421e-ac0b-58a743707e77)

I don't watch the repository, I only have it starred, thus I don't want to see announcements or anything else for this repository:

![Screenshot from 2023-09-07 09-47-37](https://github.com/community/community/assets/301255/0e2b8011-cf1d-47e3-9264-bc00c2cb72b9)

I see a clear distinction between both, and I believe a lot of people use the star as a bookmark-only feature. I want to save this repo to see later, but I don't want to follow everything that happens to it all the time. If I wanted to see announcements and other things related to the repo, then I'd watch it.

Some configuration is needed here for people that used stars as bookmarks.

Thank you.

**Comment by aerosol** [comment]
+1. I use stars to let the repo owners know I like their work (and occasionally use them as bookmarks). 
By no means I'm interested in seeing announcements for starred projects. Can't find the exact number of stars given anymore but I'm quite sure it's over a thousand. I'm watching only a handful to reduce noise.

**Reply by port19x** [reply]
I use lists to make further distinctions on why I starred something.
Is there repo owner a friend?
Do I rely on it for workflow?
Just a cool project?

Either way, the current frontpage sucks

**Reply by LunarWatcher** [reply]
You've starred 1979 repos, according to the raw HTML for the stars tab on your GH profile:
![image](https://github.com/community/community/assets/29489988/4677aeee-23a2-4ca6-9256-7eb1c3ad3ef5)

Too bad there isn't a tooltip.

~~Equally infuriating, the feed no longer seems to include stars I've gotten on my repos.~~ This seems to work again now.

Regardless, they've hit the sweet spot for moving stars as far away from their actual purpose as they can without completely redefining their scope

**Comment by fifsky** [comment]
+1  me too

**Comment by Kyogre** [comment]
Yeah, I really want to have an **option** to hide updates of starred repos in my feed.

**Comment by YSavir** [comment]
I feel the same way about this. This function actually has the unintended consequence of discouraging me from starring repos since I don't want to see them in my feed.
